scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -65,6 +68,13 @@ function doSetup() {
     if [ "${result}" = 0 ]
     then
       doAddPlugins
+    elif [ -d "${SETTINGS_PATH}"/eclipse/plugins ]
+    then
+      local file
+      for file in "${SETTINGS_PATH}"/eclipse/plugins/*.properties
+      do
+        doAddPluginFromConfig "${file}"
+      done
```

Comment:
Yes that's right, I thought `doAddPlugins` does more, but now I see that it does exactly that. `$result` and the condition thus falls away, because this is always done in `doAddPlugins`.